### PR TITLE
feat: toasts

### PR DESCRIPTION
**Pull request by nicosantangelo**
Examples:
image
image

It's functional, but we're working on some improvements for a later version

### CODE REVIEW

**Review by cazala — CHANGES_REQUESTED** [review]

File: src/components/Toast/Toast.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
+import { Close } from '../Close/Close'
+import { Props, ToastType } from './Toast.types'
+import './Toast.css'
+
+export class Toast extends React.PureComponent<Props> {
```
Comment (nicosantangelo) [inline src/components/Toast/Toast.tsx:6]: @cazala Should we export ToastType from here?

File: src/components/Toasts/Toasts.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+
+import { Props } from './Toasts.types'
+import './Toasts.css'
+
+export default class Toasts extends React.PureComponent<Props> {
```
Comment (nicosantangelo) [inline src/components/Toasts/Toasts.tsx:6]: @cazala Should we export ToastPosition from here?, to be able to import it from the root package (`decentraland-ui`)

File: src/components/Toast/Toast.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  closeTimeoutId: number | null = null
+
+  handleClose = () => {
+    this.props.onClose()
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply src/components/Toast/Toast.tsx:23]: I fixed this! but I was thinking, maybe this logic should be delegated to the user of the lib? (decentraland-dapps in our case)

I'm not sure, as maybe someone using just dcl-ui might want this behaviour, but I thought I ask

**Review by cazala — APPROVED** [review]

**Pull request merged**